### PR TITLE
[DOC] Pyroscope landing page for doc

### PR DESCRIPTION
**Pull request by knylander-grafana**
Adds a landing page for Pyroscope. 

image

Fixes:https://github.com/grafana/pyroscope/issues/3175

### CODE REVIEW

File: docs/sources/_index.md [hunk 1]
```diff
@@ -11,56 +10,55 @@ keywords:
...
   - observability
   - continuous profiling
+hero:
+  title: Grafana Pyroscope
+  level: 1
+  image: /static/img/pyroscope-logo.svg
+  width: 110
+  height: 110
+  description: >-
+    Grafana Pyroscope is an open source software project for aggregating continuous profiling data. Continuous profiling is an observability signal that allows you to understand your workload's resources usage down to the line number.
```
Comment (simonswine) [inline docs/sources/_index.md:20]: Not too sure if that makes it better or not.

```suggestion
    Grafana Pyroscope is an open source software project for aggregating continuous profiling data. Continuous profiling is an observability signal that allows you to understand your workload's resources usage down to the source code line number.
```

File: docs/sources/_index.md [hunk 2]
```diff
@@ -11,56 +10,55 @@ keywords:
...
+Grafana Pyroscope is a multi-tenant, continuous profiling aggregation system, aligning its architectural design with Grafana Mimir, Grafana Loki, and Grafana Tempo.
+It facilitates the ingestion, storage, and querying of profiles and integrates with Grafana.
+This integration enables a cohesive correlation of profiling data with existing metrics, logs, and traces.
 
+Continuous profiling provides ongoing, in-depth insights into application performance.
+With Pyroscope, you go from a bunch of flame graph files on your desktop to a database you can query and analyze production data in a structured way.
 
-<video style="border-radius: 1%" autoplay loop>
+<video style="border-radius: 1%; width: 75%; display: block; margin-left: auto; margin-right: auto;" autoplay loop>
   <source src="ui.webm" type="video/webm">
```
Comment (simonswine) [inline docs/sources/_index.md:59]: It probably makes sense to update that video. It shows a quite outdated version of explore. We probably want to show either the new explore (or the app shipped with Pyroscope).

**Review by simonswine — APPROVED** [review]
Thank you, this looks so much better. 👍

File: docs/sources/_index.md [hunk 3]
```diff
@@ -11,56 +10,55 @@ keywords:
...
 </video>
```
Comment (knylander-grafana) [inline docs/sources/_index.md:60]:
```suggestion
<!--video style="border-radius: 1%; width: 75%; display: block; margin-left: auto; margin-right: auto;" autoplay loop>
  <source src="ui.webm" type="video/webm">
</video-->
![Pyroscope UI showing the comparison view](/media/docs/pyroscope/screenshot-pyroscope-comp-view.png)
```

File: docs/sources/_index.md [hunk 4]
```diff
@@ -11,56 +10,56 @@ keywords:
...
-{{% /admonition %}}
+## Overview
 
-Pyroscope, in particular, offers the flexibility to either visualize more traditional adhoc data or evolve your applications observability tooling to include more modern continuous profiling capabilities.
+Grafana Pyroscope is a multi-tenant, continuous profiling aggregation system, aligning its architectural design with Grafana Mimir, Grafana Loki, and Grafana Tempo.
+It facilitates the ingestion, storage, and querying of profiles and integrates with Grafana.
+This integration enables a cohesive correlation of profiling data with existing metrics, logs, and traces.
 
+Continuous profiling provides ongoing, in-depth insights into application performance.
+With Pyroscope, you go from a bunch of flame graph files on your desktop to a database you can query and analyze production data in a structured way.
```
Comment (knylander-grafana) [inline docs/sources/_index.md:56]:
```suggestion
Grafana Pyroscope is a multi-tenant, continuous profiling aggregation system, aligning its architectural design with Grafana Mimir, Grafana Loki, and Grafana Tempo.
This integration enables a cohesive correlation of profiling data with existing metrics, logs, and traces.

Explore continuous profiling data to gain insights into application performance.
You can query and analyze production data in a structure way. 
Use the Pyroscope UI or Grafana to visualize the data. 
```

File: docs/sources/_index.md (hunk 2 above)
Comment (knylander-grafana) [reply docs/sources/_index.md:59]: I replace the video with this screenshot: 
image

**Pull request merged**